### PR TITLE
Fix perspective in  analysis engine

### PR DESCRIPTION
**Pull request by sanderland**
It is now always C_EMPTY due to 

- not updating the new query from rbase
- ~~not setting a default from the global config in the loadParams override~~

### CODE REVIEW

File: cpp/command/analysis.cpp [hunk 1]
```diff
@@ -704,6 +704,7 @@ int MainCmds::analysis(int argc, const char* const* argv) {
         newRequest->hist = hist;
         newRequest->nextPla = nextPla;
         newRequest->params = rbase.params;
+        newRequest->perspective = rbase.perspective;
```
Comment (lightvector) [inline cpp/command/analysis.cpp:707]: This is the only line that needs to be fixed, right?

Comment (lightvector) [reply cpp/command/analysis.cpp:707]: (that is, the other changes are unnecessary, we just need only this single line as the entire commit to fix this bug, right?)

Comment (sanderland) [reply cpp/command/analysis.cpp:707]: you're right, this was the last line I changed before thinking it was the other ones, but the cfg takes care of that. fixed.

**Comment by lightvector** [comment]
Accepted, but squashed into one commit as 3016348e.
Thanks!

**Pull request closed without merging**